depmapomics/constants.py

```diff
@@ -295,60 +293,115 @@
-    "popaf": "Popaf",
-    "likely_gof": "LikelyGoF",
-    "likely_lof": "LikelyLoF",
-    "hess_driver": "HessDriver",
-    "hess_signture": "HessSignature",
-    "hess_signature": "HessSignature",
-    "revel_score": "RevelScore",
-    "pharmgkb_id": "PharmgkbID",
-    "dida_id": "DidaID",
-    "dida_name": "DidaName",
-    "gwas_disease": "GwasDisease",
-    "gwas_pmid": "GwasPmID",
-    "gtex_gene": "GTexGene",
-    "DepMap_ID": "DepMap_ID",
+    'chrom': 'Chrom',
+    'pos': 'Pos',
+    'ref': 'Ref',
+    'alt': 'Alt',
+    'af': 'AF',
+    'dp': 'DP',
+    'ref_count': 'RefCount',
+    'alt_count': 'AltCount',
+    'gt': 'GT',
+    'ps': 'PS',
+    'variant_type': 'VariantType',
+    'variant_info': 'VariantInfo',
+    'dna_change': 'DNAChange',
+    'protein_change': 'ProteinChange',
+    'hugo_symbol': 'HugoSymbol',
+    'ensembl_gene_id': 'EnsemblGeneID',
+    'ensembl_feature_id': 'EnsemblFeatureID',
+    'hgnc_name': 'HgncName',
+    'hgnc_family': 'HgncFamily',
+    'uniprot_id': 'UniprotID',
+    'dbsnp_rs_id': 'DbsnpRsID',
+    'gc_content': 'GcContent',
+    'lof_gene_name': 'LofGeneName',
+    'lof_gene_id': 'LofGeneId',
+    'lof_number_of_transcripts_in_gene': 'LofNumberOfTranscriptsInGene',
+    'lof_percent_of_transcripts_affected': 'LofPercentOfTranscriptsAffected',
+    'nmd': 'NMD',
+    'clnsig': 'ClnSig',
+    'molecular_consequence': 'MolecularConsequence',
+    # 'af_exac': 'AfExac',
+    # 'af_esp': 'AfEsp',
+    # 'af_tgp': 'AfTgp',
+    'vep_impact': 'VepImpact',
+    'vep_biotype': 'VepBiotype',
+    'vep_hgnc_id': 'VepHgncID',
+    'vep_existing_variation': 'VepExistingVariation',
+    'vep_mane_select': 'VepManeSelect',
+    'vep_ensp': 'VepENSP',
+    'vep_swissprot': 'VepSwissprot',
+    'sift': 'Sift',
+    'polyphen': 'Polyphen',
+    'gnomade_af': 'GnomadeAF',
+    'gnomadg_af': 'GnomadgAF',
+    'vep_clin_sig': 'VepClinSig',
+    'vep_somatic': 'VepSomatic',
+    'vep_pli_gene_value': 'VepPliGeneValue',
+    'vep_loftool': 'VepLofTool',
+    # 'oncogene_high_impact': 'OncogeneHighImpact',
+    # 'tumor_suppressor_high_impact': 'TumorSuppressorHighImpact',
+    # 'achilles_top_genes': 'AchillesTopGenes',
+    'structural_relation': 'StructuralRelation',
+    'associated_with': 'AssociatedWith',
+    # 'lof': 'Lof',
+    # 'driver': 'Driver',
+    # 'likely_driver': 'LikelyDriver',
+    'transcript_likely_lof': 'TranscriptLikelyLof',
+    'brca1_func_score': 'Brca1FuncScore',
+    'civic_id': 'CivicID',
+    'civic_description': 'CivicDescription',
+    'civic_score': 'CivicScore',
+    # 'popaf': 'Popaf',
+    # 'likely_gof': 'LikelyGof',
+    'likely_lof': 'LikelyLof',
+    'hess_driver': 'HessDriver',
+    'hess_signture': 'HessSignture',
+    'revel_score': 'RevelScore',
+    'pharmgkb_id': 'PharmgkbId',
+    'dida_id': 'DidaID',
+    'dida_name': 'DidaName',
+    'gwas_disease': 'GwasDisease',
+    'gwas_pmid': 'GwasPmID',
+    'gtex_gene': 'GtexGene',
+    'cosmic_tier': 'CosmicTier',
```

Comment:
@qinqian Just to double check. I can't remember what your agreement with cosmic says... can we release it here?

Reply:
We merged cosmic with oncokb to say if it is a important cancer gene.